apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/InpatientClaimTransformerV2.java

```diff
@@ -337,12 +338,8 @@ private static ExplanationOfBenefit transformClaim(InpatientClaim claimGroup) {
           line.getDeductibleCoinsuranceCd());
 
       // HCPCS_CD => item.productOrService
-      line.getHcpcsCode()
-          .ifPresent(
-              c ->
-                  item.setProductOrService(
-                      TransformerUtilsV2.createCodeableConcept(
-                          eob, CcwCodebookVariable.HCPCS_CD, c)));
+      TransformerUtilsV2.mapHcpcs(
+          eob, item, line.getHcpcsCode(), Optional.empty(), Collections.emptyList());
```

Comment:
It's been a _day_. Can someone help me, a half-braindead person, understand this?

It looks to me like this is the only functional change in the PR? I'm guessing that, previously, HCPCS codes for this one field were being mapped in a way that's inconsistent with the rest of the application? And that inconsistency was _somehow_ causing claims with those codes to be filtered out when they shouldn't have been?

Reply:
Yeah, that's correct.  The current version of InpatientClaimTransformerV2 uses the "https://bluebutton.cms.gov/resources/variables/hcpcs_cd" coding, this coding system is unrecognized by the filter and as a [fail safe assumes that every value could be a SAMHSA code](https://github.com/CMSgov/beneficiary-fhir-data/blob/ae0b5af919d2923c5cd7b69d1f50cf28c04f5415/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4SamhsaMatcher.java#L532-L538)